src/screens/DetailScreen.js

```diff
@@ -135,8 +137,8 @@ export class DetailScreen extends React.Component {
                 {!!subtitle && <ListSubtitle>{subtitle}</ListSubtitle>}
                 {/*TODO: map multiple contentBlocks */}
                 {!!title && <ListTitle noSubtitle>{title}</ListTitle>}
-                {!!body && <HtmlView html={body} />}
-                {!!link && <Link url={link} title={'Weiterlesen'} />}
+                {!!body && <HtmlView html={trimNewLines(body)} />}
```

Comment:
trimNewLines is the new helper for trimming ? if yes it's missing in the commit :)

Reply:
I founded  trimNewLines function in htmlViewHelper !

Reply:
yes, it was already there but forgotten to use here